CHANGELOG.md

```diff
@@ -1,5 +1,132 @@
 # Changelog
 
+## [v0.4.3](https://github.com/gojek/feast/tree/v0.4.3) (2020-01-08)
+
+[Full Changelog](https://github.com/gojek/feast/compare/v0.4.2...v0.4.3)
```

Comment:
0.4.3 Changes:
- Bugfix for redis ingestion retries throwing NullPointerException on remote runners

Reply:
Updated